common/model-views.cpp

```diff
@@ -1149,6 +1149,7 @@ namespace rs2
         streaming(false), _pause(false),
         depth_colorizer(std::make_shared<rs2::gl::colorizer>()),
         yuy2rgb(std::make_shared<rs2::gl::yuy_decoder>()),
+        nv122rgb(std::make_shared<rs2::gl::nv12_decoder>()),
```

Comment:
Please replace all of `nv12` name used to `nv12i`

Reply:
Done